wlm_server/operation/models.py

```diff
@@ -1,12 +1,18 @@
 from django.db import models
 
 from user.models import User
+from channel.models import Channel
 
 class Operation(models.Model):
     user = models.ForeignKey(
         User,
         on_delete=models.PROTECT,
         related_name='operation_history',
     )
+    channel = models.ForeignKey(
```

Comment:
Shouldn't it be a Many-to-many field instead of a foreign key?

Reply:
After reconsidering following our offline discussion, to manage the operation history of a certain channel easily, it looks better to set `channel` to a foreign key.
Even with this approach, there is no issue in determining which channels the user is operating on.